data/step_function_inputs/AfriSAR_AGB_Maps_1681.json

```diff
@@ -5,5 +5,6 @@
     "discovery": "cmr",
     "asset_name": "data",
     "asset_roles": ["data"],
+    "cmr_api_url": "https://cmr.maap-project.org",
```

Comment:
Shouldn't we add this line for the other files as well?

Reply:
Since the defaults below are already changed, I don't think we need add this line here. We could also, or in addition,  add instructions for creating an environment file, like the maap.sh file that I have been using, to export environment variables:

```
export ENV=dev
export COGNITO_APP_SECRET=veda-auth-stack-maap-dev/veda-workflows
export APP_NAME="veda-data-pipelines"
export STAC_INGESTOR_API_URL=https://xxx.execute-api.us-west-2.amazonaws.com/dev/
export SECRET_NAME=maap-dev/pgstac/xxx
export DATA_MANAGEMENT_ROLE_ARN=arn:aws:iam::xxxx:role/maap-prod-uw2-lambda-processing
export CMR_API_URL=https://cmr.maap-project.org
export USER_SHARED_BUCKET=maap-user-shared-data
export BUCKET=nasa-maap-data-store
```

Reply:
Would it be too much to change the deployment to use an .env file? 😅 

But I'll go ahead and remove that cmr_api_url from the stepfn file since we modified the defaults

Reply:
@abarciauskas-bgse differing instructions from the env.sample.sh script? Could we merge as is?